Example/Database/Tests/Integration/FIRDatabaseTests.m

```diff
@@ -430,7 +430,11 @@ - (void) testCallbackQueue {
     __block BOOL done = NO;
     [database.reference.childByAutoId observeSingleEventOfType:FIRDataEventTypeValue
                                                      withBlock:^(FIRDataSnapshot *snapshot) {
-        dispatch_assert_queue(callbackQueue);
+                                                       if (@available(iOS 10.0, macOS 10.12, *)) {
```

Comment:
Just to confirm, we're good to use this since Firestore (which we test on earlier Xcode versions) doesn't need to be able to run this, correct?

Reply:
Correct.